### PR TITLE
An article about Ludwig van Beethoven Violin Sonatas

### PR DESCRIPTION
**Pull request by nadjmaakhal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,6 +32,7 @@
     <div class="column">
     </div>
     <div class="column">
+        <div id="example"><script>include("articles/music_album_review.md", "example")</script></div>
```
Comment (claudiodsf) [inline index.html]: please change the two `example` with `music_album_review` ;-)

Comment (nadjmaakhal) [reply index.html]: done !

**Comment by claudiodsf** [comment]
Looks fine now, thanks!

Squashing into a single commit and merging!

**Comment by nadjmaakhal** [comment]
thank you for the instructions

**Pull request merged**